README.md

```diff
@@ -732,3 +732,18 @@ This can be done in two ways:
 **Note:** This only configures Numba to use the current RMM resource for allocations.
 It does not initialize nor change the current resource, e.g., enabling a memory pool.
 See [here](#memoryresource-objects) for more information on changing the current memory resource.
+
+### Using RMM with PyTorch
+
+[PyTorch](https://pytorch.org/docs/stable/notes/cuda.html) can use RMM for memory allocation.
+For example, to configure PyTorch to use an RMM-managed pool, you can do the following:
+
+```python
+import rmm
+import torch
+
+rmm.reinitialize(pool_allocator=True)
+torch.cuda.memory.change_current_allocator(rmm.rmm_torch_allocator)
+```
+
+PyTorch and RMM will now share the same memory pool.
```

Comment:
What if I have a custom RMM memory resource that I want to use with PyTorch?

Reply:
Related Question, Can we use `ManagedMemoryResource ` with Pytorch and Rapids together after this PR ?

Asking cause this [PR](https://github.com/pytorch/pytorch/pull/86786) stated using Managed Memory as one of its goals and wondering if this PR will also unlock that.

Reply:
> What if I have a custom RMM memory resource that I want to use with PyTorch?

That will work. `rmm.reinitialize()` is just a convenience function to set the current device memory resource. I added yet another example showing how to use a custom MR.

> Related Question, Can we use ManagedMemoryResource with Pytorch and Rapids together after this PR ?

The example I added shows that we can do this too. :)